src/pages/home/report/ReportActionItem.js

```diff
@@ -50,6 +70,8 @@ class ReportActionItem extends Component {
     shouldComponentUpdate(nextProps, nextState) {
         return this.state.isPopoverVisible !== nextState.isPopoverVisible
             || this.props.displayAsGroup !== nextProps.displayAsGroup
+            || this.props.isMostRecentIOUReportAction !== nextProps.isMostRecentIOUReportAction
+            || !_.isEqual(this.props.report, nextProps.report)
```

Comment:
I haven't benchmarked this but subscribing to the report object in _every_ message seems potentially bad for performance since reports data changes frequently.